oxidation/libparsec/crates/client_types/tests/test_local_device.rs

```diff
@@ -232,3 +233,138 @@ fn slug(alice: &Device) {
+    //     "92bb822717b6ed29fd706250583a9007b59b87d41e072b92d78775c936a2b418cce1561b46feb8f9"
+    //     "9c2216ee8491cdf5b45f54f54cc982fee61c79c95c1b0894d8a266f2f19e5bd133352a3bf59ee160"
+    //     "4114a676272a1da1c79ada1461bcb4922d04c88eaff4022ca77e1e7cf77abd20e64e094f86438431"
+    //     "8956d44595631df703e5c93ecb4616eddd9e09f52ce551d3b842e4a3bf68731fa4d0a58a32f384eb"
+    //     "5f7273573af3b269b1033b644458eef8b6a82695edbb3daa28f7306efa3d74a0a1b217717af9d958"
+    //     "0a818c6c81d0dd33e04116b36ff62b8431070c3c7febd4405329dff739f48225d3261a5b2feac583"
+    //     "d69993577f3275a43279a59f8b783497a80043409399f93f3a7a3c6ded8ed43078f9842cb709c7db"
+    //     "221e94e620eb61943fbb5ed773ec9dd42879f82d69b01c1834eda4791d81f60254d11a9a95a388c9"
+    //     "df32630b7e5504ef361d55741c7ced33faf5249deba7b60417ed217411166954feb70da66d7bf144"
+    //     "2eae01e4f71648eefd46fb037e6a57482fdea63941a9c9159807c3b04db6d9e195942c96c7b9ae96"
+    //     "d1af2b23919e8841be9c85d5305b96149f58e020abb3aff2641370ecb2aeca9084fce184f78ff56a"
+    //     "889c9ad80e8b59e441d86c82242e70261c31cde2ba82535a5667cac313c1fb8cb0107e2b1a1f6224"
+    //     "559dc9d052be275865"
+    //   )
+    //   device_label: "My dev1 machine"
+    //   human_handle: ["alice@example.com", "Alicey McAliceFace"]
+    //   salt: hex!("6f48555e77fd45429cfe26d1dcdd3a8e")
+    &hex!(
+        "85aa63697068657274657874c5021192d7b106ad7efbbb5603a41094681bc7c65fff23066e"
+        "0e0eff8e1db5e626b626f22acd334e5ae3f992bb822717b6ed29fd706250583a9007b59b87"
+        "d41e072b92d78775c936a2b418cce1561b46feb8f99c2216ee8491cdf5b45f54f54cc982fe"
+        "e61c79c95c1b0894d8a266f2f19e5bd133352a3bf59ee1604114a676272a1da1c79ada1461"
+        "bcb4922d04c88eaff4022ca77e1e7cf77abd20e64e094f864384318956d44595631df703e5"
+        "c93ecb4616eddd9e09f52ce551d3b842e4a3bf68731fa4d0a58a32f384eb5f7273573af3b2"
+        "69b1033b644458eef8b6a82695edbb3daa28f7306efa3d74a0a1b217717af9d9580a818c6c"
+        "81d0dd33e04116b36ff62b8431070c3c7febd4405329dff739f48225d3261a5b2feac583d6"
+        "9993577f3275a43279a59f8b783497a80043409399f93f3a7a3c6ded8ed43078f9842cb709"
+        "c7db221e94e620eb61943fbb5ed773ec9dd42879f82d69b01c1834eda4791d81f60254d11a"
+        "9a95a388c9df32630b7e5504ef361d55741c7ced33faf5249deba7b60417ed217411166954"
+        "feb70da66d7bf1442eae01e4f71648eefd46fb037e6a57482fdea63941a9c9159807c3b04d"
+        "b6d9e195942c96c7b9ae96d1af2b23919e8841be9c85d5305b96149f58e020abb3aff26413"
+        "70ecb2aeca9084fce184f78ff56a889c9ad80e8b59e441d86c82242e70261c31cde2ba8253"
+        "5a5667cac313c1fb8cb0107e2b1a1f6224559dc9d052be275865ac6465766963655f6c6162"
+        "656caf4d792064657631206d616368696e65ac68756d616e5f68616e646c6592b1616c6963"
+        "65406578616d706c652e636f6db2416c69636579204d63416c69636546616365a473616c74"
+        "c4106f48555e77fd45429cfe26d1dcdd3a8ea474797065a870617373776f7264"
+    )[..], LegacyDeviceFile::Password(LegacyDeviceFilePassword {
```

Comment:
```suggestion
    )[..], 
    LegacyDeviceFile::Password(LegacyDeviceFilePassword {
```
Missing newline after `,`